### ISSUE TITLE
[Feature] Link 'blocked' issues with their blocking issue

### ISSUE THREAD

**Issue by sp4ceb4r** [rauhryan/huboard#611]
When an issue is blocked by another it would be helpful to link the blocked issue with the blocking issue.

Then, when the blocking issue is closed/resolved the blocked issue automatically unblocked.

**Comment by discorick** [comment]
Cool idea!

**Comment by benjamincharity** [comment]
Love this idea!

Also always wondered why the `blocked` and `ready for next stage` labels were handled outside the context of github labels..

**Comment by GCStokum** [comment]
Yes! I'm as addicted to Jira as Github for keeping work tidy, and that is a Very important part of workflow when you've got peeps working cross department and such.

**Comment by smiller171** [comment]
I'm wondering the same @benjamincharity

**Comment by dahlbyk** [comment]
We should definitely represent the Blocked and Ready states with GitHub labels. The challenge is how to handle that label - do we just pick a label name and create such a label if it doesn't already exist? Many projects (including [HuBoard](https://github.com/huboard/huboard/labels/BLOCKED)) already have an equivalent label, and I'm not sure we'd want to create a new, similar label in those repos.

@discorick and I have been discussing a more complete HuBoard config mechanism beyond what we've been able to achieve so far with label conventions (with some hacks like labels-as-links). Making the Blocked/Ready label names configurable (with a sensible default, of course) seems like a great use case for such a feature.

**Comment by n3m8tz** [comment]
Would love to see this

**Comment by jurajpiar** [comment]
Any news about this issue? It has been 5 years since it was opened.

**Comment by antonlvovych** [comment]
Any updates on this issue?

**Comment by PedroGFonseca** [comment]
I'm going to go ahead and assume that this is going to be permanently... blocked by something? 😇

**Comment by sp4ceb4r** [comment]
Sorry to spam mentions... @n3m8tz, @jurajpiar, @geekiam, @PedroGFonseca it would be safe to assume this project is EOL. I don't know if any of the forks of this project are actively maintained either; if one is and this hasn't been implemented there then you might suggest this feature on that fork.

**Comment by discorick** [comment]
@sp4ceb4r is correct. As a former maintainer of this project, I just haven't had the bandwidth to address issues for the past few years, and regrettably will not be in the future.

**Comment by PedroGFonseca** [comment]
@discorick @sp4ceb4r thanks for the update! (and for all your work on open sourcing the project)

**Comment by tgmpje** [comment]
Sorry to see this project gone stale. To address @sp4ceb4r's original feature request: you can use [SnoozeThis](https://www.snoozethis.com/) on GitHub to automatically unblock issues. Simply add a comment to the issue that is blocked:
> @SnoozeThis [url of issue to wait for]

SnoozeThis will then add a comment as soon as the other issue has been resolved. And by installing [the optional app](https://github.com/apps/snoozethis) you can do more advanced stuff like having SnoozeThis assign the issue back to you.
